shared/common-adapters/user-proofs.js.flow

```diff
@@ -34,4 +34,4 @@ export type Props = ({
   missingProofs: Array<MissingProof>,
 } & CommonProps)
 
-export default class Render extends React$Component<void, Props, void> {}
+export default class ProofsRender extends React$Component<void, Props, void> {}
```

Comment:
I'm not sure what our standard on this is. Maybe it should just be `Render` like most other things?